components/prodpad/actions/create-feedback/create-feedback.mjs

```diff
@@ -1,25 +1,53 @@
 import app from "../../prodpad.app.mjs";
 import utils from "../../common/utils.mjs";
+import { ConfigurationError } from "@pipedream/platform";
 
 export default {
   key: "prodpad-create-feedback",
   name: "Create Feedback",
-  description: "Creates a feedback. [See the docs](https://app.swaggerhub.com/apis-docs/ProdPad/prodpad/1.0#/Feedback/PostFeedbacks).",
+  description: "Creates feedback. [See the documentation](https://app.swaggerhub.com/apis-docs/ProdPad/prodpad/1.0#/Feedback/PostFeedbacks).",
   type: "action",
-  version: "0.0.1",
+  version: "0.0.2",
   props: {
     app,
+    feedback: {
+      propDefinition: [
+        app,
+        "feedback",
+      ],
+    },
     contactId: {
-      description: "ID of the contact providing the feedback.",
+      description: "The ID of the contact providing the feedback. Either `Contact ID` or `Contact Name` is required.",
+      optional: true,
       propDefinition: [
         app,
         "contactId",
       ],
     },
-    feedback: {
+    contactName: {
+      label: "Contact Name",
+      description: "The name of the contact. Either `Contact ID` or `Contact Name` is required.",
+      optional: true,
       propDefinition: [
         app,
-        "feedback",
+        "name",
+      ],
+    },
+    contactEmail: {
+      description: "The email of the contact. It is also used to avoid duplication of contacts and can only be filled in with `Contact Name`.",
```

Comment:
```suggestion
      description: "The email of the contact. It is also used to avoid duplication of contacts and can only be filled in with `Contact Name`."
```

Reply:
No changes needed